backend/src/main/java/com/handong/rebon/shop/domain/tag/ShopTag.java

```diff
@@ -28,4 +29,8 @@ public ShopTag(Shop shop, Tag tag) {
         this.shop = shop;
         this.tag = tag;
     }
+
+    public void deleteShopTag(){
+        this.deleteContent();
+    }
```

Comment:
커밋전 포맷팅을 적용하면 좋을 것 같네요!

```suggestion
    public void deleteShopTag() {
        this.deleteContent();
    }
```

Reply:
앗 네 감사합니다!